src/init.cpp

```diff
@@ -103,20 +105,22 @@
 #include <zmq/zmqrpc.h>
 #endif
 
+using kernel::ValidationCacheSizes;
+using node::ApplyArgsManOptions;
 using node::CacheSizes;
 using node::CalculateCacheSizes;
-using node::ChainstateLoadVerifyError;
 using node::ChainstateLoadingError;
+using node::ChainstateLoadVerifyError;
```

Comment:
In commit "validationcaches: Add and use ValidationCacheSizes" (bedef3249cfa013a48347b19fb58712a8d0fdd28)

I guess all these changes come from your editor doing case-insensitive sorts, which maybe is a preference. When I added these statements here and other places in #23497 I did purposefully use case-sensitive sorts, since I think case sensitive sorts make more sense in a case sensitive language, and look less glitchy visually, and do a better job grouping related identifiers together. You may perhaps want to keep this sorting, if it is not too much of a pain.

Reply:
I agree! Pesky editor settings... Fixed in caaa5be4ec